Disable asking/registering the parser at install time, we do this
in the INSTALL/REMOVE scripts.

--- Makefile.PL.orig	2009-04-21 19:00:26.282249017 +0200
+++ Makefile.PL	2009-04-21 19:00:33.901197702 +0200
@@ -11,43 +11,3 @@ WriteMakefile(
         'XML::NamespaceSupport' => 0.03,
     },    
 );
-
-sub MY::install {
-    package MY;
-    my $script = shift->SUPER::install(@_);
-
-    # Only modify existing ParserDetails.ini if user agrees
-
-    my $write_ini_ok = 0;
-
-    eval { require XML::SAX };
-    if ($@) {
-        $write_ini_ok = 1;
-    }
-    else {
-        my $dir = File::Basename::dirname($INC{'XML/SAX.pm'});
-        if (-e File::Spec->catfile($dir, 'SAX', 'ParserDetails.ini')) {
-            $write_ini_ok =
-                ExtUtils::MakeMaker::prompt(
-                    "Do you want XML::SAX to alter ParserDetails.ini?", "Y"
-                ) =~ /^y/i;
-        }
-        else {
-            $write_ini_ok = 1;
-        }
-    }
-    
-    if ($write_ini_ok) {
-        $script =~ s/install :: (.*)$/install :: $1 install_sax_pureperl/m;
-        $script .= <<"INSTALL";
-
-install_sax_pureperl :
-\t\@\$(PERL) -MXML::SAX -e "XML::SAX->add_parser(q(XML::SAX::PurePerl))->save_parsers()"
-
-INSTALL
-
-    }
-
-    return $script;
-}
-
